src/layouts/components/ProjectCard.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { StyledH3, StyledP, StyledUl, StyledLi } from '../../theme/globalStyle'
+import { ButtonSmall } from '../components/Button'
+
+const Wrapper = styled.div`
+  width: 400px;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
```

Comment:
Well done, @miljan-fsd !
However, I think I made a mix with the example in #33 

`Project Card` is just a *single* card. 
I put 3 cards together to illustrate that is possible to get 3 different cards from 1 component.
`CardContainer` does a function of collecting cards together.

There is no necessity to make a grid with 3 columns. One column will be enough.